lib/views/pages/profile/posts_info/info_row.dart

```diff
@@ -6,49 +6,46 @@ class InfoRow extends StatelessWidget {
   static const infoRowKey = Key("info row");
```

Comment:
I think we should use camelCase for widget keys
```suggestion
  static const infoRowKey = Key("infoRow");
```